### PR TITLE
version 1.0.4

### PR DESCRIPTION
**Pull request by Haegi**
- [x] **Flutter support**
- [x] fixed OS Transport error after closing connection
- [x] improved CI/CD
- [x] [pong with payload](https://haegi.github.io/WebSock/unknownserver_case_2_2.html)

### CODE REVIEW

**Comment by Haegi** [comment]
@Kai-Bailey could you please review & approve the PR :)

File: README.md [hunk 1]
```diff
@@ -68,7 +68,8 @@ my_server = WebSocketServer(
 my_server.serve_forever()
 ```
 
-For more guidance, check out the code for the example chat application built using WebSock in the [examples directory](https://github.com/Kai-Bailey/WebSock/tree/master/examples).
+For more guidance, check out the code for the example chat application built using WebSock in the [examples directory](https://github.com/Kai-Bailey/WebSock/tree/master/examples).   
+There you will also find the [flutter example](https://github.com/Kai-Bailey/WebSock/tree/master/examples/flutter/flutter.md).
 ## API Documentation
 You can check out the full documentation for the API [here](https://websock.readthedocs.io/en/latest/).
```
Comment (Kai-Bailey) [inline README.md]: Great work! You can add yourself as an author if you want!

**Comment by Haegi** [comment]
Thanks! Unfortunately, I don't have write access.

**Comment by Kai-Bailey** [comment]
I sent you a request to join the repo as a collaborator. It should be in your inbox!

**Pull request merged**